### ISSUE TITLE
reason for mastodon migration?

### ISSUE THREAD

**Issue by nassersaazi** [TuanaCelik/should-i-follow#5]
hi @TuanaCelik . thanks for the work being done on Haystack. i was getting around the basics of the Haystack project and landed on this repo. I would love to contribute something here though am curious...what is reason behind the mastodon migration? was it because of a change to twitter's api?

**Comment by TuanaCelik** [comment]
@nassersaazi yes indeed. The API changes have made it not possible for me to implement a working Twitter Retriever. At least from my understanding so far.
